examples/libs/ethersjs/v6/basic-transfers/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import dotenv from 'dotenv'
+import { nativeTransfer } from './transfers/native'
+import { erc20Transfer } from './transfers/erc20'
+
+dotenv.config()
+
+const main = async () => {
+  // await nativeTransfer()
```

Comment:
I would prefer to be consistent with other project and maybe have 2 different example? (even if they are both simple and look alike? 
Not really important, that's just a preference